lib/puppet_litmus/rake_tasks.rb

```diff
@@ -120,8 +120,18 @@ def run_local_command(command)
           puts "#{result['result']['node_name']}, #{args[:platform]}"
         end
       end
+    elsif args[:provisioner] == 'vagrant'
```

Comment:
This block is exactly the same as abs and vmpooler. Looks like you can also include `vagrant` on line 103. For that matter, docker could also be added there.

Reply:
That's true. good catch! Started out with the same code thinking it will end up being different...